test/e2e/cluster/cfn-templates/setup-cfn.yaml

```diff
@@ -80,7 +84,9 @@ Resources:
           Value: !Sub ${ClusterName}-cluster-vpc
         - Key: !Ref TestClusterTagKey
           Value: !Ref ClusterName
-  
+        - Key: CreationTime
```

Comment:
can we make this key a param so we can use the same constant we use in the go code when listing?